app.py

```diff
@@ -497,45 +366,50 @@ def _get_risk_level(debt_to_supply_ratio: float) -> str:
             for token in [token_1, token_2]:
                 figure = plotly.express.pie(
                     collateral_stats.reset_index(),
-                    values=f'{token} collateral',
-                    names='Protocol',
-                    title=f'{token} collateral',
+                    values=f"{token} collateral",
+                    names="Protocol",
+                    title=f"{token} collateral",
                     color_discrete_sequence=plotly.express.colors.sequential.Oranges_r,
                 )
                 streamlit.plotly_chart(figure, True)
             for token in [token_1, token_2]:
                 figure = plotly.express.pie(
                     debt_stats.reset_index(),
-                    values=f'{token} debt',
-                    names='Protocol',
-                    title=f'{token} debt',
+                    values=f"{token} debt",
+                    names="Protocol",
+                    title=f"{token} debt",
                     color_discrete_sequence=plotly.express.colors.sequential.Greens_r,
                 )
                 streamlit.plotly_chart(figure, True)
             for token in [token_1, token_2]:
                 figure = plotly.express.pie(
                     supply_stats.reset_index(),
-                    values=f'{token} supply',
-                    names='Protocol',
-                    title=f'{token} supply',
+                    values=f"{token} supply",
+                    names="Protocol",
+                    title=f"{token} supply",
                     color_discrete_sequence=plotly.express.colors.sequential.Blues_r,
                 )
                 streamlit.plotly_chart(figure, True)
 
-    last_update = src.persistent_state.load_pickle(path=src.persistent_state.LAST_UPDATE_FILENAME)
+    last_update = src.persistent_state.load_pickle(
+        path=src.persistent_state.LAST_UPDATE_FILENAME
+    )
     last_timestamp = last_update["timestamp"]
     last_block_number = last_update["block_number"]
     date_str = datetime.datetime.utcfromtimestamp(int(last_timestamp))
     streamlit.write(f"Last updated {date_str} UTC, last block: {last_block_number}.")
 
 
 if __name__ == "__main__":
-    logging.basicConfig(level=logging.INFO, format='%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+    logging.basicConfig(
+        level=logging.INFO,
+        format="%(asctime)s - %(name)s - %(levelname)s - %(message)s",
+    )
 
     streamlit.set_page_config(
         layout="wide",
         page_title="DeRisk by Carmine Finance",
         page_icon="https://carmine.finance/assets/logo.svg",
     )
 
-    main()
\ No newline at end of file
+    main()
```

Comment:
Isn;t the PR missing some changes? Seeing `charts.py` in Oleh's comments. Also, the issue setup contains this 
![image](https://github.com/user-attachments/assets/b5f2848d-954f-4b71-8fae-f6d401917283)